erdpy_network/network_config.py

```diff
@@ -1,11 +1,12 @@
-from typing import Dict, Any
+from typing import Any, Dict
 
 
 class NetworkConfig:
     def __init__(self) -> None:
         self.chain_id: str = 'T'
         self.gas_per_data_byte: int = 1500
         self.top_up_factor: float = 0
+        self.start_time: int = 0
```

Comment:
there are some additional fields that aren't included here, for example `erd_max_gas_per_transaction` (which I think can be useful)
[https://gateway.elrond.com/network/config](https://gateway.elrond.com/network/config)

Reply:
Noted, thank you :pray: